halo/evmstaking2/keeper/evmstaking2.proto

```diff
@@ -7,8 +7,8 @@ import "octane/evmengine/types/tx.proto";
 
 option go_package = "halo/evmstaking2/keeper";
```

Comment:
I'd rename the proto file to `evmstaking.proto` since then we don't need to rename later. The name might be included in some underlying identifiers.